packages/scheduler/src/SchedulerFeatureFlags.js

```diff
@@ -8,7 +8,7 @@
 
 export const enableSchedulerDebugging = false;
 export const enableIsInputPending = false;
-export const enableProfiling = false;
+export const enableProfiling = __VARIANT__;
```

Comment:
Also this change should ensure we leave the feature off for OSS builds while still running tests to make sure it doesn't break FB builds.

Reply:
Yes

Reply:
Enable profiling